app/views/test/telephony/index.html.erb

```diff
@@ -4,16 +4,16 @@
 
 <h1>Outbound calls and messages</h1>
 
+<%= button_to 'Clear messages and calls', test_telephony_path, method: :delete, class: 'usa-button' %>
 
-<div class="card card-wide">
-  <div class="padding-4">
-    <%= button_to 'Clear messages and calls', test_telephony_path, method: :delete, class: 'usa-button' %>
-    <div class="grid-row grid-gap margin-top-2">
-      <div class="tablet:grid-col-6">
-        <h2>Messages</h2>
+<div class="grid-row grid-gap margin-top-2 flex-align-stretch">
+  <% [['Messages', @messages], ['Calls', @calls]].each do |title, messages| %>
```

Comment:
I started to panic about the un-translated text but then realized this is an internal test page